pkg/blobplugin/main.go

```diff
@@ -38,6 +38,9 @@ func init() {
 var (
 	endpoint                               = flag.String("endpoint", "unix://tmp/csi.sock", "CSI endpoint")
 	blobfuseProxyEndpoint                  = flag.String("blobfuse-proxy-endpoint", "unix://tmp/blobfuse-proxy.sock", "blobfuse-proxy endpoint")
+	edgeCacheConfigEndpoint                = flag.String("edge-cache-config-endpoint", "hydrad:8675", "edge cache config endpoint")
```

Comment:
can this reference a different service (ex. hydra-config)?

Reply:
fixed in latest update